lua/sg/cody/state.lua

```diff
@@ -61,8 +61,9 @@ end
 --- Get a new completion, based on the state
 ---@param bufnr number
 ---@param win number
----@param callback function(noti)
-function State:complete(bufnr, win, callback)
+---@param code_only boolean
+---@param callback CodyChatCallback
+function State:complete(bufnr, win, code_only, callback)
```

Comment:
i think `code_only` should be `opts` and we can have a field for `code_only`. Cause we'll probably add more stuff for this later.